src/journeys_utils.js

```diff
@@ -258,7 +270,6 @@ journeys_utils.addIframeOuterCSS = function() {
 		var calculatedBodyMargin = +bannerMarginNumber + bodyMarginBottomNumber;
 		bodyMargin = 'margin-bottom: ' + calculatedBodyMargin + 'px';
 	}
-
 	// adds margin to the parent of div being inserted into
 	if (journeys_utils.divToInjectParents && journeys_utils.divToInjectParents.length > 0) {
```

Comment:
We can remove the first check here since we are setting a default on line 132. It will have a value, so this is unnecesary.